### ISSUE TITLE
OOBEEULA error after debloat

### ISSUE THREAD

**Issue by JVerhulst** [Sycnex/Windows10Debloater#86]
So I'm building a debloated Windows Image for my WDS server, but after a debloat and making the image ready with sysprep, I uploaded it to de WDS server. 

After that I did a install of the image., but the installation hangs on a OOBEEULA error.

So I was thinking if someone had dealth with this before? Or has a solution for this issue.

In the mean time I will search further for an other solution.

**Comment by Sycnex** [comment]
Did you use the Windows10SysPrepDebloater.ps1? I'm not aware of it affect the OOBEEULA but perhaps there is something that is killing it. Do you have a picture of the error that I can look at?

**Comment by JVerhulst** [comment]
No I used the "Windows10GuiDebloater.ps1" [https://imgur.com/a/pDezEsZ](url) but I added a regkey that would dissable that.

But I used the "bloatware remove with blacklist" and "without blacklist", so that I could boot and didn't get the error that the windows installations is broken, but now I get this error.


Hopefully you have a solution for me?

**Comment by Sycnex** [comment]
What registry key did you add? Is it one that touches OOBE? My scripts don’t touch OOBE since it can leave windows being non functional.

**Comment by JVerhulst** [comment]
Yeah that registry key does something with oobe, it was a fix on the microsoft forum where it would let me get pass that point where im stuck now.

This is wat he said about the reg key: "I find this workaround:

After image deployment, at the OOBE wizard, I import this registry  key to skip user OOBE:

[HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Setup\OOBE]

"UnattendCreatedUser"=dword:00000001

So only computer OOBE run"

Here is the [link](https://social.technet.microsoft.com/Forums/en-US/efdc1e5a-ffa5-421f-8f6c-a1c63a52a469/windows-10-oobe-loop-after-sysprep?forum=win10itprosetup)

But it didn't fix it for me, so I hope you may know a solution?
Maybe you can edit the "sysprep" powershell tool so that I can get the windows store (and more) out of my windows image or give me a explanation to add more windows porgrams.

**Comment by Sycnex** [comment]
Interesting. I can comment out the sysprep code so that we can check to see if it happens on any other machines that you deploy it with?

Try giving this a shot: https://www.tenforums.com/windows-insider/118225-insider-isos-cant-get-past-oobe-stage.html

**Comment by JVerhulst** [comment]
Yes it's indeed interesting, maybe because I delete the store or some other build in app that has caused this issue, I found a workaround for it at the moment and that's a answer file that I builded that accepts/skips the EULA and creates a user, so when deploying the image it's usable.

I would like to try it on a other system but I'm building it in a virtualized environment :sweat_smile:

Sure I'll look at that forum post maybe that can be of some help.

**Comment by Sycnex** [comment]
Okay let me know!

**Comment by jtmv** [comment]
Hi,

I think this can help people who are looking for a solution regarding this problem…
When the error “OOBEEULA” is displayed, do the following:

– Open a command prompt with Shift+F10
– Open registry editor: regedit
– Go to HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\Setup\OOBE
– Add a DWORD value named “SetupDisplayedEula” with the data “1”
– Press “retry” on the OOBE screen… and the EULA step is passed. 😉

**Comment by Sycnex** [comment]
Hi @JVerhulst - Is this still an issue that is happening? I am following up on old issues.

**Comment by JVerhulst** [comment]
hi @Sycnex I'm sorry to inform you a bit late but I stopped working on the project. So unless someone else has this issue, you can close it.

**Comment by nkumar2171** [comment]
This may help you - <a href="https://www.howto-connect.com/fix-oobeeula-error-something-went-wrong-windows-10-or-11/">Fix OOBEEULA Error Something went wrong in Windows 10 or 11</a>
